be/src/geo/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ add_library(Geo STATIC
     ${GENSRC_DIR}/geo/wkt_lex.l.cpp
     ${GENSRC_DIR}/geo/wkt_yacc.y.cpp
 )
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
there is nothing wrong to disable the specific error here. However, instead of tuning the compile flags scattered in separated cmakelists, I'd suggest do it in the top level to centralize the configuration for the entire project.